openstack/neutron-hypervisor-agents/templates/configmap-etc-vpod.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-etc-vpod
+  labels:
+    system: openstack
+    type: configuration
+    component: neutron
+data:
+  segment-names: |
+    {{- range $v := .Values.aci.aci_hostgroups.hostgroups }}
+    {{ $v.name }}
+    {{- end }}
+  {{- $vpods_conf := default (dict) .Values.vpods_conf }}
+  vpod.conf: |
+    {{- include "util.helpers.valuesToIni" $vpods_conf | nindent 6 }}
```

Comment:
minor: I guess `nindent 4` would template yaml in the style we usually do :D

Reply:
Changed it.